src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -417,16 +418,43 @@ public void cannotCheckOutWhenItemIsAlreadyCheckedOut() {
   }
 
   @Test
-  public void cannotCheckOutWhenItemIsMissing() {
+  public void canCheckOutMissingItem() {
     final IndividualResource missingItem = setupMissingItem(itemsFixture);
     final IndividualResource steve = usersFixture.steve();
 
-    final Response response = loansFixture.attemptCheckOutByBarcode(
+    final IndividualResource response = loansFixture.checkOutByBarcode(
       missingItem, steve);
 
-    assertThat(response.getJson(), hasErrorWith(allOf(
-      hasMessageContaining("has the item status Missing"),
-      hasItemBarcodeParameter(missingItem))));
+    assertThat(response.getJson(), hasJsonPath("status.name", "Open"));
+    assertThat(response.getJson(), hasJsonPath("borrower", notNullValue()));
+    assertThat(response.getJson(), hasJsonPath("item", notNullValue()));
+    assertThat(response.getJson(), hasJsonPath("item.status.name", CHECKED_OUT));
+    assertThat(response.getJson(), hasJsonPath("itemId", missingItem.getId().toString()));
+
+    assertThat(itemsClient.getById(missingItem.getId()).getJson(),
+      hasJsonPath("status.name", CHECKED_OUT));
+  }
+
+  @Test
+  public void canCheckOutMissingDiscoverySuppressedItem() {
```

Comment:
Why is it relevant if an item is suppressed from discovery or not?

Reply:
The jira states that if it is suppressed from discovery then the flag should be remaining after check-out. So I've added a test to verify.

Reply:
I don't understand why that is important. 

Please can you ask the PO for this story why that aspect is important (please reference me from the comment)?

Reply:
Here is the note on jira:

![image](https://user-images.githubusercontent.com/9094650/77086428-739b5b80-6a0a-11ea-8cc4-e65baf9cca1d.png)

I've added a test to verify it.

Reply:
@bohdan-suprun Thanks. I think it is important to understand why that requirement is present, to determine if it justifies an API in mod-circulation.

Does that make sense?

I can ask the question if you would prefer?

Reply:
@marcjohnson-kint, I'll appreciate if you ask the question. Thank you!

Reply:
Ok, I've asked the question

Reply:
Emma has responded. 

Personally, I think based upon her comments, this isn't an important enough aspect of this change (from an API perspective) to warrant testing it, because it is a remark rather than an expectation and unlikely to occur unless changes are made to change this part of the item record.

What do you think?

Reply:
Thanks @marcjohnson-kint, will remove the test for it.